posts/2023-07-20_cross_company_dev/cross_industry_dev.qmd

```diff
@@ -0,0 +1,148 @@
+---
+title: "Cross-Industry Open Source Package Development"
+author:
+  - name: Stefan Thoma
+description: ""
+date: "2023-07-25"
+categories: [community, admiral]
+image: "Rbasel.png"
+editor: 
+  markdown: 
+    wrap: sentence
+---
+
+<!--------------- my typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2023-07-20_shiny_run_script"
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+This post is based on [a talk](https://stefanthoma.github.io/useR_2023_admiral/#/title-slide) given at the [regional useR! conference on July 21st 2023 in Basel](https://user-regional-2023.gitlab.io/basel/).
+I took the opportunity to present my personal perspective on the current cross-industry package development efforts with a particular focus on the transformation of the job description of statistical programmers. 
+As I have only recently started my position at Roche, my personal perspective is the perspective of a newcomer. 
+I have a background in Psychology and Statistics and joined Roche in November 2022 as an intern switching to a permanent position as a statistical programmer -- what they call analytical data scientist now -- in April 2023.
+I spend about 20% of my time in such a cross-industry package development project, which was a major reason for applying for this position. 
+In this post I would like to explain how we work in this project, and why this had such an impact on my decision to join Roche. 
+
+## Context
+
+My decision was influenced by two current industry trends: 
+
+First, the switch to a more language agnostic and open source approach for clinical reporting and data analysis. 
+At the moment, R seems to be the best fitting tool for the job, but the systems used here are language agnostic in general. 
+This is related, but does not necessarily lead to the second trend: 
+The move toward cross-industry collaboration when developing clinical reporting software. 
+
+As the industry moves toward new (to them) programming languages, fit-for-purpose tools need(ed) to be developed.
+The realization that siloed solutions -- in an area where competitiveness does not benefit patients -- are simply resource hungry ways to solve the same problem in parallel provided a great argument for a shift toward collaborations. 
+
+Such cross-industry collaborations gave rise to the pharmaverse, an opinionated collection of R packages designed to solve clinical reporting in R.
```

Comment:
@StefanThoma sorry for late input - just catching up after hols. would be good to switch the word "opinionated" to "curated" if possible for pharmaverse please. we tried to drop opinionated on our site as it confused people when we offer numerous table packages for examples